### DISCUSSION TITLE
Discord Plugin Doesn't Work

### DISCUSSION THREAD

**Discussion by NaturesButler** [pear-devs/pear-desktop#908] [Q&A]
The app never connects to discord, and when I try to reconnect it either one of these messages pops up every time.
(I know there is a post like this from a month ago, but I figured my situation might be different because of the different os)

![YouTube_Music_bKQkVUMTyc](https://user-images.githubusercontent.com/119908682/205792645-45f2d909-bcf4-4c37-8476-9f2a20f104df.png)
![YouTube_Music_aUkDMwPYjJ](https://user-images.githubusercontent.com/119908682/205792647-531c6b7d-5b6f-4df0-9399-a6ba5d634599.png)

**Comment by Zo-Bro-23** [comment]
This is a known issue. Do a custom build to fix it. If you're on Windows, here's how you do a custom build (from my answer to [this](https://github.com/th-ch/youtube-music/issues/895#issuecomment-1327121950) issue):
- Download and install [NodeJS](https://nodejs.org/en/) (make sure add to path is selected)
- Download [this](https://github.com/th-ch/youtube-music/archive/3605e32b2531efa3d124ef509ac2612347bf4679.zip) zip file
- Unzip it (extract all)
- Open the extracted folder (should contain many files)
- Right click the folder in Explorer and select Open Command Prompt Here (or Windows Terminal)
- Type the following one by one:
   - `npm i -g yarn`
   - `yarn`
   - `npm i rimraf`
   - `yarn build:win`
- Wait a while
- There should be a setup file created in ./dist/
- Use that to install the app

@th-ch can we have a special release for this one? A lot of people seem to be facing the same issue.

**Reply by NaturesButler** [reply]
Thanks this worked great, in the mean time I also figured out why a different script wasn't working!

**Reply by lividcreature** [reply]
I know this is old but just wanted to say thank you! Now I can show off my awesome taste in music to all the randos I do not know.
